transformers/tokenization_gpt2.py

```diff
@@ -237,3 +237,36 @@ def save_vocabulary(self, save_directory):
                 index += 1
 
         return vocab_file, merge_file
+
+class GPT2TokenizerFast(FastPreTrainedTokenizer):
+    vocab_files_names = VOCAB_FILES_NAMES
+    pretrained_vocab_files_map = PRETRAINED_VOCAB_FILES_MAP
+    max_model_input_sizes = PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES
+
+    def __init__(self, vocab_file, merges_file, unk_token="<|endoftext|>", bos_token="<|endoftext|>",
+                 eos_token="<|endoftext|>", pad_to_max_length=False, add_prefix_space=True,
+                 max_length=None, stride=0, truncation_strategy='longest_first', **kwargs):
+
+        try:
+            from tokenizers import Tokenizer, models, pre_tokenizers, decoders
+
+            super(GPT2TokenizerFast, self).__init__(bos_token=bos_token, eos_token=eos_token, unk_token=unk_token, **kwargs)
+
+            self._tokenizer = Tokenizer(models.BPE.from_files(vocab_file, merges_file))
```

Comment:
Finding a way to get inline documentation in the `tokenizers` module is a must IMO (maybe through `.pyi` files?)

Reply:
Yes, that would be really nice. It can be more tricky than expected sometimes (pytorch is still struggling with that...). Do you want to have a look maybe? Otherwise, I can try to dive into it.

Reply:
I'll look into it!